vlog4j-core/pom.xml

```diff
@@ -60,5 +61,54 @@
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<id>client</id>
+			<dependencies>
+				<dependency>
+					<groupId>${project.groupId}</groupId>
+					<artifactId>vlog4j-client</artifactId>

```

Comment:
I need to look into this, it seems that we have a cyclic dependency. Let's discuss soon.

Reply:
This does not even work with mvn install -Pclient. 
Why is this profile not declared in vlog4j-client, (or in parent), but in vlog4j-core?

Reply:
This is strange. I think I don't have a cyclic dependency in the code. Also, mvn install -Pclient work for me. Yes, let's discuss about it soon, :)
I had placed `client` in beside `vlog-core` because `development` was there. I've moved it to parent.